src/components/NavBar/NavBar.tsx

```diff
@@ -25,7 +25,8 @@ const NavBar = () => {
   const userEmailId = localStorage.getItem("userEmailId");
 
   return (
-    <Navbar bg="dark" variant="dark" expand="lg" expanded={expanded} style={{ userSelect: 'none' }}>
+    <Navbar bg="dark" variant="dark" expand="lg" expanded={expanded} style={{ userSelect: 'none' }}
```

Comment:
Add React Bootstrap's Position Utility to make the navbar sticky.
 
Use this as reference: [React Bootstrap Navbar](https://react-bootstrap.netlify.app/docs/components/navbar/#sticky-top)